src/Compilers/CSharp/Portable/Syntax/LambdaUtilities.cs

```diff
@@ -216,12 +216,12 @@ private static bool IsReducedSelectOrGroupByClause(SelectOrGroupClauseSyntax sel
                 return false;
             }
 
-            var selectorIdentifier = ((IdentifierNameSyntax)selectOrGroupExpression).Identifier;
+            SyntaxToken selectorIdentifier = ((IdentifierNameSyntax)selectOrGroupExpression).Identifier;
 
             SyntaxToken sourceIdentifier;
             QueryBodySyntax containingBody;
 
-            var containingQueryOrContinuation = selectOrGroupClause.Parent.Parent;
+            CSharpSyntaxNode containingQueryOrContinuation = selectOrGroupClause.Parent.Parent;
```

Comment:
❓ How did this happen?

Reply:
Hum, actually it builds fine from VS and with `msbuild /v:m /m Compilers.sln`. But I get the same error as CI when I run `build.cmd`... That's mysterious

Reply:
Turns out this file is linked into `CSharpFeatures.csproj` too, and that's where it failed (in VS and first command-line, I was only building `Compilers.sln`).

Reply:
> Turns out this file is linked into CSharpFeatures.csproj too, and that's where it failed (in VS and first command-line, I was only building Compilers.sln).

Interesting.  Question:

Did you only have Compilers.sln open in VS when you did the fix?  Or did you have Roslyn.sln open?  If the latter, then we should open a bug on this.

Thanks!

Reply:
I did it with only Compilers.sln open.

Reply:
Ah ok.